style.css

```diff
@@ -0,0 +1,650 @@
+  flex-direction: column;
+  background-image: url(images/main-bg.png);
+  background-size: 100%;
+  padding: 20px 5%;
+}
+
+h1 {
+  margin: 0;
+  font-size: 37px;
+  color: #ec5242;
+}
+
+h3 {
+  margin-bottom: 0;
+  margin-top: 5%;
+  color: #ec5242;
+}
+
+p {
+  display: flex;
+  margin-bottom: 0;
+  border: 2px solid white;
+  height: 135px;
+  align-items: center;
+  margin-top: 20px;
+  background-color: #f7f7f7;
+}
+
+h2 {
+  margin-bottom: 0;
+  font-size: 26px;
+  font-weight: 100;
+  margin-top: 15px;
+}
+
+h4 {
+  margin-top: 6px;
+  font-weight: 100;
+  margin-bottom: 8%;
+}
+
+#program {
+  background-image: url('images/patter-bg.png');
+  color: white;
+  padding: 3%;
+  display: grid;
+  grid-auto-columns: 1fr;
+  justify-items: center;
+}
+
+.program-title {
+  color: white;
+  text-align: center;
+  padding-top: 24px;
+  padding-bottom: 20px;
+}
+
+.text-speaker p {
+  background: 0;
+  border: 0;
+  text-align: unset;
+  height: 0;
+  margin-top: 10px;
+}
+
+.about p {
+  border: 1px solid;
+  background-color: white;
+  margin-top: 30px;
+}
+
+#program p {
+  background: 0;
+  border: 0;
+  height: 0;
+  text-align: unset;
+}
+
+.line-small {
+  height: 1px;
+  width: 34px;
+  background-color: #ec5242;
+  margin-bottom: 20px;
+}
+
+.program-box {
+  font-size: 14px;
+  display: flex;
+  flex-direction: column;
+}
+
+.lecture {
+  display: grid;
+  grid-template-columns: 40% 54%;
+  justify-content: space-between;
+  background-color: rgba(136, 136, 136, 0.3);
+  padding: 20px;
+  margin-bottom: 15px;
+}
+
+.program-left-text {
+  display: flex;
+  color: #ec5242;
+  gap: 10px;
+  font-size: 20px;
+  font-weight: 700;
+  align-items: center;
+}
+
+.icons {
+  width: 40px;
+  margin-right: 8px;
+}
+
+.program-button {
+  background-color: #ec5242;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  max-width: 300px;
+  height: 80px;
+  margin-bottom: 35px;
+  padding: 30px 30px;
+  text-align: center;
+  margin-top: 35px;
+  cursor: pointer;
+}
+
+.chess-speaker1 {
+  width: 40%;
+  height: 40%;
+}
+
+.speaker-content {
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  gap: 10px;
+  margin: 10px;
+}
+
+.text-speaker h3 {
+  color: black;
+  margin-left: 0;
+}
+
+.second-p {
+  margin-top: 55px;
+  color: black;
+}
+
+.text-speaker {
+  font-size: 13px;
+  color: #ec5242;
+}
+
+.speaker-main {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.speakerLine {
+  height: 1px;
+  width: 34px;
+  background-color: #ec5242;
+  margin-bottom: 40px;
+  margin-left: 46%;
+  margin-top: 5%;
+}
+
+.speakers-title {
+  text-align: center;
+  color: black;
+  font-size: 25px;
+  margin-top: 45px;
+  margin-bottom: 30px;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  background-repeat: no-repeat;
+  padding: 20px 5%;
+  background-position-y: 17%;
+  background-origin: content-box;
+  background-color: #f7f7f7;
+  margin-top: 5%;
```

Comment:
- [ ] Please remove this `margin-top: 5%;` from your about section, it is adding a blank space at top of your page for the mobile view.